Detectors/FIT/simulation/src/DigitizerTask.cxx

```diff
@@ -78,11 +86,40 @@ void DigitizerTask::FinishTask()
   if (!mContinuous)
     return;
 
+  // is there QED backgroung provided? Fill QED slots up to the end of reserved ROFrames
+  if (mQEDBranch) {
+    Float_t digitROtime = 25; //ns
+    processQEDBackground(digitROtime);
```

Comment:
While I still did not get the details of the readout, this line is certainly wrong: ```processQEDBackground``` takes argument of the max time stamp to fill by the QED contribution and does it for all ```mQEDEntryTimeBinNS```-long time intervals between the last processed ```mLastQEDTimeNS``` time and provided tMax.

Reply:
Hello
FIT will read each event each 25ns, does not matter central or ultra-peripheral. QED electrons are not background for us. Events are not mixed in readout in continuous mode. If 2 events happened within 25 ns, FIT will register 2 times and 1 common amplitude and set flag that it was 2 events. digitROtime really = 25ns. 
Best regards
Alla

Reply:
@AllaMaevskaya : if you need the QED not as a background, then current implementation will not work: it adds the QED electrons integrated during the readout of the detector on top of the hadronic interaction provided to whole Alice. Therefore it ignores multiple bunch crossings w/o hadronic interaction but with a huge amount of QED ones. 
To have it as a signal, the QED should come as a standard (and dominant) component of the cocktail. but given its huge x-section (~200kb) this means multiple (~50) Geant triggers per ever BC (while the hadronic interaction prob. is 0.002 per BC). This can work for special productions, but I guess for the routine MC we should consider the QED as a background.